src/dune

```diff
@@ -0,0 +1,5 @@
+(executable
+ (name extension)
```

Comment:
I'd rename this executable to `vscode-ocaml-platform`. The name `extension` is a bit too generic.

Reply:
If I change the name to `vscode-ocaml-platform`, dune looks for the `Vscode-ocaml-platform` module as an entry point. That doesn't seem like a very optimal change.

Is there a way to have separate names for the executable and entry-point module?